runtime/builtin_types_test.go

```diff
@@ -23,6 +23,33 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		fooObject := newTestClass("Foo", []*Type{ObjectType}, toDictUnsafe(args[0])).ToObject()
+		result, raised := builtinDelAttr(f, append(Args{fooObject}, args[1:]...), nil)
+		if raised != nil || result != None {
+			return result, raised
+		}
+		result, raised = GetAttr(f, fooObject, toStrUnsafe(args[1]), nil)
+		if raised != nil || result != nil {
+			return None, raised
```

Comment:
I think it's preferable to return an exception only when the first return value is nil. Therefore, I would just `return GetAttr(...)`. That will be simpler anyway.